src/delivery-options/data/request/requestData.js

```diff
@@ -35,6 +35,6 @@ const parametersByPlatform = {
 export function getParametersByPlatform(platform = realConfigBus.get(CONFIG.PLATFORM), configBus = realConfigBus) {
   return {
     ...parametersByPlatform[platform](configBus),
-    cutoff_time: setCutOffTime(configBus),
+    cutoff_time: getCutOffTime(configBus),
```

Comment:
Oh, j'ai confused. Why did this need to change?

Reply:
It doesn't set, it gets!